### PR TITLE
Added classes for outcomes user progress

### PR DESCRIPTION
**Pull request by rgoessl**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -304,20 +304,31 @@ export const Classes = {
 		selected: 'selected'
 	},
 	outcomes: {
+		assessed: 'assessed',
+		demonstration: 'demonstration',
+		demonstratableLevel: 'demonstratable-level',
 		intent: 'intent',
 		intents: 'intents',
 		intentList: 'intent-list',
+		loa: 'level-of-achievment',
```
Comment (CodeBaboon) [inline index.js]:
```suggestion
		levelOfAchievement: 'level-of-achievment',
```

File: index.js [hunk 2]
```diff
@@ -304,20 +304,31 @@ export const Classes = {
...
 	meetings: {
 		bookable: 'bookable'
 	},
-	relativeUri: 'relative-uri'
+	relativeUri: 'relative-uri',
+	userProgress: {
+		outcomes: {
+			root: 'user-progress-outcomes',
+			outcome: 'user-progress-outcome',
+			activities: 'outcome-activities',
```
Comment (CodeBaboon) [inline index.js]: Should this be either moved up to the more general `outcomes` class or else the value modified to be `user-progress-outcome-activities`?

Comment (rgoessl) [reply index.js]: The latter

File: index.js [hunk 3]
```diff
@@ -304,20 +304,31 @@ export const Classes = {
...
+		suggested: 'suggested'
 	},
 	meetings: {
 		bookable: 'bookable'
 	},
-	relativeUri: 'relative-uri'
+	relativeUri: 'relative-uri',
+	userProgress: {
+		outcomes: {
+			root: 'user-progress-outcomes',
```
Comment (CodeBaboon) [inline index.js]: I think changing from `root` to `outcomes` would be more in line with the existing similar implementations (including the general case `outcomes.outcomes` from above).

```suggestion
			outcomes: 'user-progress-outcomes',
```

File: index.js (hunk 2 above)
Comment (CodeBaboon) [reply index.js]:
```suggestion
			activities: 'user-progress-outcome-activities',
```

**Pull request merged**